puppet/zulip/manifests/app_frontend_once.pp

```diff
@@ -60,4 +60,12 @@
     minute => '22',
   }
 
+  # 1st day every 3 months.
+  zulip::cron { 'update-stream-active-status':
+    minute => '0',
+    hour   => '0',
+    day    => '1',
+    month  => '*/3',
```

Comment:
Wait, we're running this every 3 months?  That seems ... _extremely_ chunky.  Why would we want to run it that infrequently?  That makes every run much more heavy-weight, and makes the user experience worse.